src/app/utils/formValidationUtils.js

```diff
@@ -156,19 +143,19 @@ function validate(form, cb) {
       validate: () => true,
       errorMsg: '',
     },
-    'starting-page': {
+    startPage: {
       validate: (val) => isNumeric(val),
       errorMsg: 'Please enter the starting page',
     },
-    'ending-page': {
+    endPage: {
       validate: (val) => isNumeric(val),
       errorMsg: 'Please enter the end page',
     },
   };
 
   const error = {};
   _mapObject(form, (val, key) => {
-    const isValid = fieldsToCheck[key].validate(val);
+    const isValid = fieldsToCheck[key].validate('' + val);
```

Comment:
https://www.npmjs.com/package/validator
The library we are using now specifically says that it validates and sanitizes strings only or it will throw an error.
It still check number and other formats I think but after the value has been converted into a string.